app/assets/stylesheets/active_admin/components/_comments.scss

```
@@ -4,7 +4,7 @@
   .active_admin_comment {
     clear: both;
     margin-top: 10px;
-    margin-bottom: 40px;
+    margin-bottom: 50px;
```

We can lower this value to just be a nice, consistent spacing between comment containers.
```suggestion
    margin-bottom: 20px;
```
Since the containers were not properly cleared beforehand, this wasn't having the right effect as shown by the screenshots shared.